homeassistant/components/sensibo/climate.py

```diff
@@ -188,6 +192,20 @@ def _do_update(self, data):
             if key in FIELD_TO_FLAG:
                 self._supported_features |= FIELD_TO_FLAG[key]
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device information about this entity."""
+        return {
```

Comment:
Create an instance of DeviceInfo instead
```suggestion
        return DeviceInfo(
```